api/src/main/resources/prod.db.properties

```diff
@@ -0,0 +1,6 @@
+application.gifts.datasource.driverClassName=org.postgresql.Driver
```

Comment:
looks like these properties are redundant and should be replaced with `spring.datasource.` ones